test/e2e/pages/allocation-details.js

```diff
@@ -0,0 +1,44 @@
+import { endOfWeek, format } from 'date-fns'
+import { Selector } from 'testcafe'
+
+import { fillInForm } from '../_helpers'
+
+import Page from './page'
+
+class AllocationDetails extends Page {
+  constructor() {
+    super()
+    this.url = '/allocation/new/allocation-details'
+    this.fields = {
+      movesCount: Selector('#moves_count'),
+      fromLocation: Selector('#from_location'),
+      toLocation: Selector('#to_location'),
+      date: Selector('#date'),
+    }
+  }
+
+  async fill() {
+    const fieldsToFill = [
+      {
+        selector: this.fields.movesCount,
+        value: '3',
+      },
+      {
+        selector: this.fields.fromLocation,
+        type: 'autocomplete',
+        value: 0,
```

Comment:
I think it don't pass a value it will use a random value which I think would be better to test with.

Reply:
This is something that I wanted to discuss.  For some reason, the test fails intermittently if the chosen option is not immediately visible. This makes the test reliable, however is clearly not a permanent solution.

Reply:
Ah okay. This feels like the issue we have seen elsewhere. Can we add a `TODO` around this and a little description of why we're always selecting the first item. It will help our future selves and future developers if somebody else picks up this codebase.